File: dataset_reduzido/avatar/Python/Code/codeforces_313_B.py
soz = input ( )
a = [ 0 ] * ( len ( soz ) + 1 )
for i in range ( 1 , len ( soz ) ) :
    a [ i ] = a [ i - 1 ]
    if soz [ i - 1 ] == soz [ i ] :
        a [ i ] += 1
n = int ( input ( ) )
arr = [ ]
for i in range ( n ) :
    m , l = input ( ).split ( )
    arr.append ( a [ int ( l ) - 1 ] - a [ int ( m ) - 1 ] )
for i in arr :
    print ( i )

File: output/StarCoder-100/avatar/Python/Java/codeforces_313_B.java
import java.util.Scanner;

public class codeforces_313_B {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String soz = sc.next();
        int[] a = new int[soz.length() + 1];
        for (int i = 1; i < soz.length(); i++) {
            a[i] = a[i - 1];
            if (soz.charAt(i - 1) == soz.charAt(i)) {
                a[i] += 1;
            }
        }
        int n = sc.nextInt();
        int[] arr = new int[n];
        for (int i = 0; i < n; i++) {
            String[] m_l = sc.next().split(" ");
            arr[i] = a[Integer.parseInt(l) - 1] - a[Integer.parseInt(m) - 1];
        }
        for (int i = 0; i < n; i++) {
            System.out.println(arr[i]);
        }
    }
}
